libtorch-ffi/src/Torch/Internal/Unmanaged/Type/Module.hs

```diff
@@ -85,27 +87,86 @@ train obj on = [C.throwBlock| void {
     );
   }|]
 
-run_method :: Ptr Module -> Ptr StdString -> Ptr (C10List IValue) -> IO (Ptr IValue)
-run_method obj method_name args = [C.throwBlock| at::IValue* {
+runMethod :: Ptr Module -> Ptr StdString -> Ptr (C10List IValue) -> IO (Ptr IValue)
+runMethod obj method_name args = [C.throwBlock| at::IValue* {
     return new at::IValue($(torch::jit::script::Module* obj)->run_method(
       *$(std::string* method_name)
     , *$(c10::List<at::IValue>* args)
     ));
   }|]
 
-run_method1 :: Ptr Module -> Ptr StdString -> Ptr IValue -> IO (Ptr IValue)
-run_method1 obj method_name args = [C.throwBlock| at::IValue* {
+runMethod1 :: Ptr Module -> Ptr StdString -> Ptr IValue -> IO (Ptr IValue)
+runMethod1 obj method_name args = [C.throwBlock| at::IValue* {
     return new at::IValue($(torch::jit::script::Module* obj)->run_method(
       *$(std::string* method_name)
     , *$(at::IValue* args)
     ));
   }|]
 
+getParameters :: Ptr Module -> IO (Ptr TensorList)
+getParameters obj = [C.throwBlock| std::vector<at::Tensor>* {
+    std::vector<at::Tensor>* vec_parameters = new std::vector<at::Tensor>();
+    auto parameters = $(torch::jit::script::Module* obj)->parameters();
+    for(auto p : parameters) {
+      vec_parameters->push_back(p);
+    }
+    return vec_parameters;
+  }|]
+
+
+setParameters :: Ptr Module -> Ptr TensorList -> IO ()
+setParameters obj params = [C.throwBlock| void {
+    auto parameters = $(torch::jit::script::Module* obj)->parameters();
```

Comment:
This paremeters  are just copy.
https://github.com/pytorch/pytorch/blob/v1.4.1/torch/csrc/jit/script/module.h#L295